index.html

```diff
@@ -2271,11 +2272,11 @@ <h3>Terms of Use</h3>
           <dd>
 The value of the <code>termsOfUse</code> <a>property</a> MUST
 specify one or more terms of use policies under which the creator issued the
-<a>credential</a> or <a>presentation</a>. If the recipient (a <a>holder</a> or
-<a>verifier</a>) is not willing to adhere to the specified terms of use, then
-they do so on their own responsibility and might incur legal liability if they
-violate the stated terms of use. Each <code>termsOfUse</code> value MUST
-specify its <a>type</a>, for example, <code>IssuerPolicy</code>, and
+<a>verifiable credential</a> or <a>verifiable presentation</a>. If the recipient
```

Comment:
nope. strike verifiable for both changes. We meant the former.

Reply:
Fixed.